kratos/utilities/matrix_vector_utilities.h

```diff
@@ -0,0 +1,51 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics 
+//
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    
+//                    
+//
+
+
+#if !defined(KRATOS_MATRIX_VECTOR_UTILITIES_INCLUDED )
+#define  KRATOS_MATRIX_VECTOR_UTILITIES_INCLUDED
+
+// System includes
+
+// External includes
+
+// Project includes
```

Comment:
This is the only thing that I dislike, maybe to specific, and better to include this sort of things in the future AMatrix instead of a standalone utility

Reply:
I agree with doing it in AMatrix in the future. I think this is the most boiler plate code in kratos. It appears in like +1 million places and I just wanted to make it disappear...

Reply:
I also don't think this should be hidden in a function.